tests/core/objects/test_dataset_type.py

```diff
@@ -250,3 +250,13 @@ def test_bytes_type():
     assert bdt.serialize(b) == b
     assert bdt.deserialize(b) == b
     assert bdt.requirements.modules == []
+
+
+# def test_all_have_writers():
```

Comment:
No tests yet?

Reply:
it appears that for now there are to many dataset types that we cant implement writers for now (.ext mainly). I implemented them for builtin types and also for pandas/numpy which will cover 90% of non-DL stuff

Reply:
What are the problems with implementing other dataset types?